app/helpers/ops_helper.rb

```diff
@@ -3,6 +3,51 @@ module OpsHelper
   include_concern 'MyServer'
   include VmwareConsoleHelper
 
+  def row_data(label, value)
+    {:cells => {:label => label, :value => value}}
+  end
+
+  def database_details
+    @database_details = ActiveRecord::Base.configurations[Rails.env]
+    @database_display_name =
+      if @database_details["host"].in?([nil, "", "localhost", "127.0.0.1"])
+        _("Internal Database")
+      else
+        _("External Database")
+      end
+      @data = {:title => _('Basic Information')}
+      @data[:rows] = [
+        row_data(_('Name'), @database_display_name),
+        row_data(_('Hostname'), @database_details["host"]),
+        row_data(_('Database name'), @database_details["database"]),
+        row_data(_('Username'), @database_details["username"])
+      ]
```

Comment:
style looks weird here - think this part should un-indented.

Reply:
Oh yeah the bot [mentioned it](https://github.com/ManageIQ/manageiq-ui-classic/pull/8546#issuecomment-1344856695), but it was buried in the middle of the comment

I'm wondering if we should get rid of that `Rails/HelperInstanceVariable` since it seems to be in the way more than it helps. 🤔

Reply:
instance variables in partial files will also give us a notification...